.github/workflows/build.yml

```diff
@@ -10,9 +10,9 @@ jobs:
     strategy:
       matrix:
         logstash: [
-          { version: '7.17.4', url: "https://artifacts.elastic.co/downloads/logstash/logstash-7.17.4-linux-x86_64.tar.gz" , main: 'true' },
-          { version: '6.8.23', url: "https://artifacts.elastic.co/downloads/logstash/logstash-6.8.23.tar.gz" , main: 'false' },
-          { version: '8.2.2', url: "https://artifacts.elastic.co/downloads/logstash/logstash-8.2.2-linux-x86_64.tar.gz" , main: 'false' }
+          #{ version: '7.17.4', url: "https://artifacts.elastic.co/downloads/logstash/logstash-7.17.4-linux-x86_64.tar.gz" , main: 'true' },
```

Comment:
We are no longer building on the old versions?

Reply:
The latest version seems to be 8.10. Challenges with old LS versions is that they use outdated Jackson and other dependent libraries causing failures in installation and runs

Reply:
Instead of commenting out, just remove everything not supported.